docs/api-reference-source/npm.xml

```diff
@@ -37,10 +37,10 @@ cli.tell('Total count of unique friend names: ' + uniqueFriendNames.length);
 				<p>
 					To access a package with special characters in its name, use bracket notation.
 				</p>
-			
-				<df-sample name="Use the uuid package">
-const uuid = npm['uuid/v4'];
-cli.tell('New unique identifier: ' + uuid());
+				
+				<df-sample name="Use the Octokit package">
+const octokit = new npm['@octokit/rest'].Octokit();
+let myRepos = await octokit.rest.repos.listForUser({ username: myName }));
```

Comment:
And you updated the docs too, perfect 😊

Here I'd like to see if we can make this example even simpler. In all the samples, I try to have as little complexity as possible, as little stuff detracting from what's being illustrated—here, for instance, I'm worried someone looking at the code could wonder whether or not the `new` operator is part of the npm module syntax. If you can find a scoped package that doesn't need instantiating, it'll make the sample even clearer.

Similarly, it'd be better if the actual call to the package was simpler (the whole “await octokit.rest.repos…” thing), but that's a lot less important; I don't expect it to cause confusion, it just adds to the sample size.

Reply:
This was harder than expected. A lot of packages that are going through the trouble of scoping include multiple types that are intended to be instantiated. I found a derivation of the uuid package (staying on the theme, I guess). What do you think of that one?

I've been using [all-the-package-names](https://www.npmjs.com/package/all-the-packages) `| grep '@' | less` and randomly picking packages to try... :)